sql/core/src/main/scala/org/apache/spark/sql/execution/subquery.scala

```diff
@@ -171,6 +173,68 @@ case class InSubqueryExec(
   }
 }
 
+/**
+ * The physical node of exists-subquery. This is for support use exists in join's on condition,
```

Comment:
We can just say `The physical node for non-correlated EXISTS subquery.` This is pretty general and not only for join conditions